### PR TITLE
Bug 1622800 - part 6: Let l10n-screenshotis.sh pass `--test-without-building` to fastlane

### PR DESCRIPTION
**Pull request by JohanLorenzo**
*Thank you for submitting a pull request, your contributions are greatly appreciated!*

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Please make sure you've run the test scheme and all the tests pass. If your changes affect existing tests please make sure to update the tests as well.

### CODE REVIEW

**Comment by JohanLorenzo** [comment]
I've made some progress. I'd like to get this change in the script landed so I can pursue my testing. I'm currently not 100% sure the `--test-without-building` works the way I want it too. So I need further testing 🙂

File: l10n-screenshots.sh [hunk 1]
```diff
@@ -25,5 +32,6 @@ for lang in $LOCALES; do
         --derived_data_path l10n-screenshots-dd \
         --erase_simulator --localize_simulator \
         --devices "iPhone 8" --languages "$lang" \
-        --output_directory "l10n-screenshots/$lang" > "l10n-screenshots/$lang/snapshot.log" 2>&1
```
Comment (isabelrios) [inline l10n-screenshots.sh:28]: @JohanLorenzo removing this partt: `"l10n-screenshots/$lang/snapshot.log"` we may be missing the log file? It has been very helpful to look at it while debugging issues when launching these tests...

Comment (isabelrios) [reply l10n-screenshots.sh:28]: Sure, you are right, makes sense to have the logs visible in Bitrise somehow. Thanks for the explanation!

**Comment by JohanLorenzo** [comment]
Great! I'm going to merge this PR so I can make some progress on #6411 today. Feel free to drop your review @garvankeeley. I'll be happy to address your comments in a followup 🙂

**Pull request merged**